api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimService.java

```diff
@@ -55,8 +56,14 @@ public ClaimResult createOrUpdateClaim(ClaimRequest claimRequest) {
 
         try {
             EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateClaimant(claimRequest.getClaimant());
-            if (claimExistsAndIsEligible(decision)) {
+            if (claimExists(decision)) {
                 Claim claim = findClaim(decision.getExistingClaimId());
+
+                if (decisionIsNotEligible(decision)) {
```

Comment:
This is the wrong place to invalidate a claim - we want to change the status to pending_expiry if the eligibility check for the payment_cycle reports ineligible, and leave the existing claim alone if a new, ineligible claim is received for that nino.